book/src/SUMMARY.md

```diff
@@ -22,7 +22,7 @@
   - [Staking Delegation and Rewards](stake-delegation-and-rewards.md)
   - [Performance Metrics](performance-metrics.md)
 
-- [Anatomy of a Fullnode](fullnode.md)
+- [Anatomy of a Validator](fullnode.md)
```

Comment:
Change the filename too?

Reply:
hmm, after renaming validator.md to validator-proposed.md perhaps?

Reply:
✔️